keylime/tenant.py

```diff
@@ -387,7 +387,7 @@ def check_ek(self,ek,ekcert,tpm):
 
     def validate_tpm_quote(self,public_key,quote,tpm_version,hash_alg):
         registrar_client.init_client_tls(config,'tenant')
-        reg_keys = registrar_client.getKeys(self.cloudverifier_ip,self.registrar_port,self.agent_uuid)
+        reg_keys = registrar_client.getKeys(self.registrar_ip,self.registrar_port,self.agent_uuid)
```

Comment:
Brilliant catch, we would never have noticed this from running everything over local host.